EDU-3009: Update Global Admin references to include new Account Owner role (#3154)
temporalio/documentation: docs/production-deployment/cloud/get-started.mdx

id: get-started
title: Get started with Temporal Cloud
sidebar_label: Get started

> description: Start using Temporal Cloud by signing up, having the Account Owner or Global Admin role, setting up CA certificates, creating a Namespace, inviting users, and connecting your Temporal Client and Workers.

Author: ```suggestion
description: Get started with Temporal Cloud. Sign up, verify your Account Owner or Global Admin role, set up CA certificates, create a Namespace, invite users, and connect your Temporal Client and Workers.
```
